rest-service/manager_rest/rest/resources_v2_1/deployment_update.py

```diff
@@ -188,12 +191,15 @@ def _parse_args(deployment_id, request_json, using_post_request=False):
             'auto_correct_types',
             request_json.get('auto_correct_types', False)
         )
+        reevaluate_active_statuses = verify_and_convert_bool(
+            'reevaluate_active_statuses',
+            request_json.get('reevaluate_active_statuses', False)
+        )
         force = verify_and_convert_bool(
             'force',
             request_json.get('force', False)
         )
         manager = get_deployment_updates_manager(preview)
-        manager.validate_no_active_updates_per_deployment(deployment_id)
```

Comment:
It was kind of weird to validate it inside `_parse_args` function.  Moved to `_commit()` and `manager.stage_deployment_update()`